docs/tutorial/web-embeds.md

```diff
@@ -0,0 +1,22 @@
+# Web embeds in Electron
+
+If you want to embed (third party) web content in an Electron `BrowserWindow`, there are three options available to you: `<iframe>` tags, `<webview>` tags, and `BrowserViews`. Each one offers slightly different functionality and is useful in different situations. To help you choose between these, this guide will explain the differences and capabilities of each.
+
+## Iframes
+
+Iframes in Electron behave like iframes in regular browsers. An `<iframe>` element in your page can show external web pages, provided that their [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) allows it. To limit the amount of capabilities a site in an `<iframe>` tag, it's recommended to use the [`sandbox` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox) and only allow the capabilities you want to support.
+
+## Webviews
```

Comment:
```suggestion
## WebViews
```

Reply:
We don't name them "WebViews" anywhere (actually we only name them lowercased: https://electronjs.org/docs/api/webview-tag but that felt weird for a heading)

Reply:
Oh, I see I do that in the line below. :sweat_smile:  I'll commit this :)